csharp-advanced-topics/ActionFuncInCSharp/new 3.txt

```diff
@@ -0,0 +1,25 @@
+nuget.exe push -Source "CAT-OS_Feed" -ApiKey az "C:\Users\IgorKorolenko\source\repos\CAT-OS-Project\*.nupkg" --skip-duplicate
```

Comment:
What is this file for?

Reply:
This should be excluded. It's for other project.